src/PresentationalComponents/Snippets/ErrorHandler.js

```diff
@@ -47,14 +50,16 @@ const ErrorHandler = ({ code, ErrorState, EmptyState }) => {
         default:
             return ErrorState && <ErrorState />
                 || EmptyState && <EmptyState />
+                || !metadata.has_systems && <NoRegisteredSystems />
```

Comment:
you are accessing `metadata` object which is not marked as required in propTypes. if it is not passed, there must be an error. please, use `!metadata?.has_systems` or `metadata: propTypes.object.isRequired`